modules/node-observability-run-profiling-query.adoc

```diff
@@ -6,7 +6,7 @@
 [id="running-profiling-query_{context}"]
 = Running profiling query
 
-Profiling query is a blocking operation that fetches CRI-O and Kubelet profiling data for a duration of 30 seconds. The Node Observability Operator stores the profiling data inside the container file system `/run/node-observability` directory. To request profiling data query, you have to create a `NodeObservabilityRun` resource.
+To run profiling query, you must create a `NodeObservabilityRun` resource. The profiling query is a blocking operation that fetches CRI-O and Kubelet profiling data for a duration of 30 seconds. When the profiling query is complete, you must retrieve the profiling data inside the container file system `/run/node-observability` directory.
```

Comment:
>To run profiling query

Should this be "a profiling query" or "the profiling query"?

>When the profiling query is complete

Suggest "After the profiling..."